river/multioutput/encoder.py

```diff
@@ -1,35 +1,37 @@
+from __future__ import annotations
```

Comment:
Note: when you add this to the top of the file, the `pre-commit` logic will automatically convert your type annotations (thanks @kulbachcedric!)